dist/overflow-color.umd.js

```diff
@@ -33,19 +33,7 @@
   var setBgColor = function setBgColor(color) {
     if (currentBgColor !== color) {
       currentBgColor = color;
-      var css = 'html { background: ' + currentBgColor + '; }';
-
-      if (!styleTag) {
-        styleTag = document.createElement('style');
-        var head = document.head || document.getElementsByTagName('head')[0];
-        head.appendChild(styleTag);
-      }
-
-      if (styleTag.styleSheet) {
-        styleTag.styleSheet.cssText = css;
-      } else {
-        styleTag.innerHTML = css;
-      }
+      document.documentElement.style.backgroundColor = color;
```

Comment:
Hey @toptal-yorikim: could you document with comments in the code each of these changes that you're making? Just reading through this code, it makes it really hard to understand the purpose of each of these changes, which means that this would be hard for us to maintain going forward

Reply:
Ok. Well in this function most of the code seems obsolete and redundant, and more of that, a bit slowly too. It was confirmed when some white flickers were appearing on the page without scroll, it just didn't have enough time to change the color. So It makes sense to apply `background-color` to the root element of the page in this way, because we want it to flood the whole viewport and change it fast on scrolling.